docs/advanced/security.rst

```diff
@@ -0,0 +1,55 @@
+=================================
+Cluster Security & Authentication
+=================================
+
+A Flocker cluster comprises a control service and convergence agents, along with some command line tools that are provided to interact with, and manage the cluster. For more information, see :doc:`architecture`.
+
+Flocker uses `Transport Layer Security <http://en.wikipedia.org/wiki/Transport_Layer_Security>`_ (TLS) to authenticate components of a cluster, in a `mutual authentication  <http://en.wikipedia.org/wiki/Mutual_authentication>`_ model.
+
+This ensures that the control service, convergence agents, and API end users are communicating with a verified component of a cluster, helping to prevent unauthorised access, and mitigating some potential attack vectors.
+
+Client Certification Overview
+=============================
+
+When :doc:`installing the flocker-node package <../indepth/installation>`, the ``flocker-ca`` tool (provided as part of ``flocker-cli``) generates a certificate authority for your cluster, as well as certificates and private keys for the control service and convergence agents.
+
+A certification authority certificate, also known as a root certificate, is a type of TLS certificate that is used to generate other certificates, by signing another key's `certificate signing request <http://en.wikipedia.org/wiki/Certificate_signing_request>`_ with its own private key.
+The certificate authority is an individual or organisation that is trusted to generate certificates signed with its own private key.
+In the context of Flocker, the certificate authority is the cluster administrator.
+
+This mutual authentication process, where certificates signed by a private key known only to the cluster administrator, are used to establish connections and to provide a means to verify the identity of an entity, such as the control service or an API end user, which presents itself as being an authorised component of a cluster.
+
+For example, if a convergence agent receives a request to change the state of a cluster node from a machine identifying itself as that cluster's control service, it can check that the TLS data received was encrypted using the key of a certificate signed by the cluster's known certificate authority.
```

Comment:
This is overcomplex, and I don't think you need to explain the basics of public key cryptography. What you do need to explain is what the administrator needs to do at high level, and what the various components of the software need to do. So in this bit you can say that the server validates client certificate was signed by the CA, and client validates server certificate was signed by the CA.